ArchiSteamFarm/Steam/Storage/BotConfig.cs

```diff
@@ -580,6 +580,21 @@ public sealed class BotConfig {
 				return (null, null);
 			}
 
+			if (botConfig.PasswordFormat == ArchiCryptoHelper.ECryptoMethod.PlainText) {
+				ASF.ArchiLogger.LogGenericWarning(string.Format(CultureInfo.CurrentCulture, Strings.WarningPasswordSavedAsPlaintext, nameof(Steam)));
+			}
+
+			if (botConfig.IsSteamPasswordSet && botConfig.DecryptedSteamPassword != null) {
+				string[] disallowedValues = botConfig.IsSteamLoginSet switch {
```

Comment:
Once you have `ISet<string>` above, this could be written in a cleaner way with `HashSet<string> disallowedValues = new() { "account" }` with `if` for adding account name.